src/app/public/modules/radio/radio.component.ts

```diff
@@ -126,6 +172,11 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
     this.changeDetector.detectChanges();
   }
 
+/**
+ * Specifies and binds a value to the radio button's value property. The value usually corresponds
```

Comment:
Yup, Alex recommended revising it. His comments are below.
>While this may be true, I feel like we're missing the most important part of this definition. I would suggest we be a bit more specific. This property is binded to the radio button's value property. If you look at W3school's definition for that corresponding value property, it states:

>The value attribute defines the unique value associated with each radio button. The value is not shown to the user, but is the value that is sent to the server on "submit" to identify which radio button that was selected.

To your point though, I think the second "value" in that first sentence should be code snippet.

Reply:
Sounds good. I assumed the change was an improvement but just wanted to make sure. I've had a few copy/paste miscues when setting up phase 2 docs, so just wanted to double check.

Reply:
✔️